### PR TITLE
Deploy terraform from GitHub

### PR DESCRIPTION
**Pull request by russdaygh**

### CODE REVIEW

File: PocketDDD.BlazorClient/PocketDDD.BlazorClient/wwwroot/appsettings.dev.json
Comment (dynamictulip) [inline PocketDDD.BlazorClient/PocketDDD.BlazorClient/wwwroot/appsettings.dev.json:1]: `Development` is the default env when running locally on your computer, we shouldn't rename this one

File: PocketDDD.BlazorClient/PocketDDD.BlazorClient/wwwroot/appsettings.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "apiUrl": "",
-  "fakeBackend": false
+  "fakeBackend": true
```
Comment (dynamictulip) [inline PocketDDD.BlazorClient/PocketDDD.BlazorClient/wwwroot/appsettings.json]: I don't think we should change this, by default I think it is better to blow up if the server has not been configured is unreachable. For local development we can use the `Development` appsettings

**Review by dynamictulip — COMMENTED** [review]
So awesome!
Have made a comment but otherwise looks good to me :-)

File: PocketDDD.BlazorClient/PocketDDD.BlazorClient/wwwroot/appsettings.dev.json
Comment (russdaygh) [reply PocketDDD.BlazorClient/PocketDDD.BlazorClient/wwwroot/appsettings.dev.json:1]: Thanks, I've reverted that. I was having trouble getting settings working properly and forgot to revert. Same as the other change.

**Review by dynamictulip — APPROVED** [review]

**Pull request merged**